core/src/main/scala/org/apache/spark/api/python/PythonWorkerFactory.scala

```diff
@@ -395,12 +395,16 @@ private[spark] class PythonWorkerFactory(
     }
   }
 
+  private val workerLogCapture =
+    envVars.get("SPARK_SESSION_UUID").map(new PythonWorkerLogCapture(_))
```

Comment:
SPARK_SESSION_UUID looks like too general name if this is supposed to be used for only PySpark environment. Do you think we can have `PYSPARK_` prefix like the other PySpark environment variables, @ueshin ?

```
PYSPARK_PYTHON
PYSPARK_DRIVER_PYTHON
PYSPARK_ARCHIVES_PATH
PYSPARK_EXECUTOR_MEMORY_MB
PYSPARK_PLANNER_MEMORY_MB
PYSPARK_SUBMIT_ARGS
PYSPARK_PIN_THREAD
PYSPARK_NO_NAMESPACE_SHARE
PYSPARK_UDS_MODE
```